storage/client/queue/src/main/java/com/azure/storage/queue/Utility.java

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.storage.queue;
+
+import com.azure.core.implementation.util.ImplUtils;
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+import java.util.Locale;
+import java.util.TreeMap;
+
+
+public final class Utility {
```

Comment:
If this is being copied from my PR to add a common package for storage make this package private as it will be replaced in the future.

Same comment for the public methods in this class.

Reply:
Good call. Done.